src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -50,6 +50,9 @@ function LtiConfigForm({
       <Form ref={formRef} onSubmit={handleSubmit}>
         <h3 className="mb-3">{title}</h3>
         <p>{intl.formatMessage(messages.formInstructions)}</p>
+        {app.messages && Object.values(app.messages).flat().map(msg => (
+          <p key={msg}>{msg}</p>
```

Comment:
do you think passing `msg` as key is right here? can we use anything else? 
e.g. `{provider-name}-messages-{idx}`

Reply:
Ok, got it.